### ISSUE TITLE
Add 2019 participant quotes

### ISSUE THREAD

**Issue by emiliom** [oceanhackweek/oceanhackweek.github.io#70]
The front page on the web site has [two participant quotes from 2018 only](https://oceanhackweek.github.io/#three). We could add to that with two 2019 participants, from [this UW eScience news item](https://escience.washington.edu/oceanhackweek-participants-engage-in-brainstorm-of-data-techniques/); or replace the 2018 ones.

**Comment by ocefpaf** [comment]
B/c we have only two I say we add them. That would be 5 in total and they are not too long.

**Comment by emiliom** [comment]
I'll copy here the OHW19 quotes.

This one is easy b/c it's brief:

**Ágata Piffer Braga** - My experience at the Oceanhackweek 2019 was in a few words a once in a lifetime learning experience. The people involved were amazing, from the organization group to all the participants.

The project group I joined was extremely democratic and involved. I learned not only from more advanced researchers but also telling what I knew to my group partners. I definitely recommend this event for everyone that is eager to learn.

The other one is fairly long. I'll paste it here, but I think we'd probably want to trim it to make it shorter.

**Xu Chen** - I feel so lucky that I didn’t miss the brainstorm of the data techniques in OHW2019. I have been using python for data analysis about two years since I realized its simplicity, beauty, and powerfulness, especially comparing to other languages I used before. And I was quite satisfied with Python packages, such as NumPy and Matplotlib, which are extensively used in my data analysis for numerical model outputs. So when I heard about Oceanhackweek, I expected that probably I could meet more Python fellows in the ocean community, and it will be very beneficial to stay in a community for debugging and learn new staff in future works.

Actually, the week spent in OHW2019 paid me much more than I could ever have expected. Packages, such as xarray and Dask, expanded my vision on python’s applications in ocean data analysis. Lectures on Github opened the door of so many repositories of open-source projects. Pangeo provided a cutting-edge platform for oceanographers to use packages specifically written for ocean data. Cloud data storage and cloud computation demonstrated great potential for ocean researches.

And the project work during the workshop really makes our hands dirty in using the techniques we learned. Our team worked on quality control using machine-learning methods. It was a lot of cooperation, hardworking, fun, and friendships. Our presentation showed good results and we decided to keep working on it after the workshop. Last but not least, Seattle was really beautiful in late August and UW provided a very nice service of lodging; I had a very pleasant experience living in the dorm with my nice roommate.
